api/src/main/java/org/pmiops/workbench/compliancetraining/ComplianceTrainingServiceImpl.java

```diff
@@ -108,13 +123,8 @@ public boolean useAbsorb() {
    * @param agent
    */
   @Transactional
-  public DbUser syncComplianceTrainingStatus(DbUser dbUser, Agent agent)
+  public DbUser syncComplianceTrainingStatusMoodle(DbUser dbUser, Agent agent)
```

Comment:
There should be no functional changes to the Moodle flow in this PR. The changes here are refactors to share code with Absorb.